jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/HttpRequestFactory.java

```diff
@@ -5,6 +5,8 @@
 import com.yahoo.jdisc.http.HttpRequest;
 import com.yahoo.jdisc.http.servlet.ServletRequest;
 import com.yahoo.jdisc.service.CurrentContainer;
+import org.eclipse.jetty.util.Utf8Appendable;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Unnecessary import, and I think the convention is to use the javax variant anyway.

Reply:
That imported was not intended. It's was inserted by the IDE during an automatic refactoring. I have now removed it and updated the commit.